website/docs/reference/global-configs.md

```diff
@@ -179,61 +197,59 @@ See [structured logging](events-logging#structured-logging) for more details.
 
 :::
 
-
 </File>
 
-## Writing JSON artifacts
-
-The `WRITE_JSON` config determines whether dbt writes JSON artifacts (eg. `manifest.json`, `run_results.json`) to the `target/` directory. JSON serialization can be slow, and turning this flag off _might_ make invocations of dbt faster. Alternatively, you might disable this config if you want to perform a dbt operation and avoid overwriting artifacts from a previous run step.
-
-<File name='Usage'>
+### Partial Parsing
 
-```text
-$ dbt --no-write-json run
-```
+The `PARTIAL_PARSE` config can turn partial parsing on or off in your project. See [the docs on parsing](parsing#partial-parsing) for more details.
 
-</File>
+<File name='profiles.yml'>
 
-## Strict
+```yaml
 
-As of v1.0, the `-S` or `--strict` flag has been deprecated.
+config:
+  partial_parse: true
 
-## Warnings as Errors
+```
 
-Turning on the `WARN_ERROR` config will convert dbt warnings into errors. Any time dbt would normally warn, it will instead raise an error. Examples include `--select` criteria that selects no resources, deprecations, configurations with no associated models, invalid test configurations, or tests and freshness checks that are configured to return warnings.
+</File>
 
 <File name='Usage'>
 
 ```text
-$ dbt --warn-error run
-...
+dbt --no-partial-parse run
 ```
 
 </File>
 
-## Partial Parsing
-The `PARTIAL_PARSE` config can turn partial parsing on or off in your project. See [the docs on parsing](parsing#partial-parsing) for more details.
+### Printer width
+
+By default, dbt will print out lines padded to 80 characters wide. You can change this setting by adding the following to your `profiles.yml` file:
 
 <File name='profiles.yml'>
 
 ```yaml
-
 config:
-  partial_parse: true
-
+  printer_width: 120
 ```
 
 </File>
 
-<File name='Usage'>
+### Send anonymous usage stats
```

Comment:
One update here: We just merged a community contribution that supports an env var named `DO_NOT_TRACK`, which will turn on/off the same config: https://github.com/dbt-labs/dbt-core/pull/5000

These are equivalent:
- `DO_NOT_TRACK=1` is the same as `DBT_SEND_ANONYMOUS_USAGE_STATS=False
- `DO_NOT_TRACK=2` is the same as `DBT_SEND_ANONYMOUS_USAGE_STATS=True`

Reply:
Added this in https://github.com/dbt-labs/docs.getdbt.com/pull/1319/commits/af4adbe7e555faf316fb6b9fa174a8419d18bc5e